tests/integration_tests/config/validate_file_using_schema_json_input.yml

```diff
@@ -0,0 +1,7 @@
+CTA_SIMPIPE:
+  APPLICATION: simtools-validate-file-using-schema
+  TEST_NAME: yml_input
+  CONFIGURATION:
+    SCHEMA: >
+      https://raw.githubusercontent.com/gammasim/workflows/main/schemas/altitude.schema.yml
```

Comment:
should we add a second test to test `.yaml` extension or would that be overkill?

Reply:
Ok, then unless strictly necessary, we could stop supporting `.yaml` and remove the if statement in the read function that deals with it.